projects/igniteui-angular/migrations/update-12_0_0/changes/inputs.json

```diff
@@ -0,0 +1,61 @@
+{
+    "@scheme": "../../common/schema/binding.schema.json",
+    "changes": [
+        {
+            "name": "tabsType",
```

Comment:
Not sure, but maybe we should rename tabsType to tabAlignment.

Reply:
The switch from old tabsType to tabAlignment is done by the custom migrations. This InputChange is to remove the tabsType from igx-tabs component(since it has been migrated already with the custom migration).
Custom migrations is necessary since the tabsType(fixed,contentFit) values are not as the same as tabAlignment(start, justify)